CMakeLists.txt

```diff
@@ -111,107 +155,96 @@ elseif(NOT WIN32)
 	set(WITH_LIBBACKTRACE TRUE)
 endif()
 
-include_directories("include")
-include_directories("src")
-
-file(GLOB_RECURSE SENTRY_NATIVE_TEST_SOURCES
-	${CMAKE_CURRENT_SOURCE_DIR}/tests/unit/*.c
+target_include_directories(sentry PUBLIC
+	"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>"
+	"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>"
+	"$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
 
 if(LINUX)
-	set(LINK_LIBRARIES ${LINK_LIBRARIES} "pthread" "dl")
+	target_link_libraries(sentry PRIVATE pthread dl)
 endif()
 
 if(WITH_LIBUNWINDSTACK)
-	include_directories(${CMAKE_CURRENT_SOURCE_DIR}/external/libunwindstack-ndk/include)
-	add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/external/libunwindstack-ndk/cmake)
-	set(LINK_LIBRARIES ${LINK_LIBRARIES} "unwindstack")
+	target_include_directories(sentry PRIVATE
+		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/libunwindstack-ndk/include)>")
+	add_subdirectory(${PROJECT_SOURCE_DIR}/external/libunwindstack-ndk/cmake)
+	target_link_libraries(sentry PRIVATE unwindstack)
+	if(NOT BUILD_SHARED_LIBS)
+		sentry_install(TARGETS unwindstack EXPORT sentry
+			LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+			ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+		)
+	endif()
 endif()
 
 if(WIN32)
-	set(LINK_LIBRARIES ${LINK_LIBRARIES} "dbghelp.lib" "pathcch.lib")
+	target_link_libraries(sentry PRIVATE dbghelp pathcch)
 endif()
 
-SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
-SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${PROJECT_BINARY_DIR})
-SET(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${PROJECT_BINARY_DIR})
-SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
-SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG ${PROJECT_BINARY_DIR})
-SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE ${PROJECT_BINARY_DIR})
-SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR})
-SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG ${PROJECT_BINARY_DIR})
-SET(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE ${PROJECT_BINARY_DIR})
-
-# builtin `list(TRANSFORM)` (https://cmake.org/cmake/help/latest/command/list.html#transform)
-# was only added in cmake 3.12, but we target 3.10, so we define a custom
-# functions which can prepend a prefix to all list elements.
-# copied from https://stackoverflow.com/a/27630120
-FUNCTION(PREPEND var prefix)
-	SET(listVar "")
-	FOREACH(f ${ARGN})
-		LIST(APPEND listVar "${prefix}/${f}")
-	ENDFOREACH(f)
-	SET(${var} "${listVar}" PARENT_SCOPE)
-ENDFUNCTION(PREPEND)
-
 if(BACKEND_CRASHPAD)
 	add_subdirectory(external/crashpad EXCLUDE_FROM_ALL)
-	include_directories(external/crashpad external/crashpad/third_party/mini_chromium/mini_chromium)
-	add_definitions(-DSENTRY_WITH_CRASHPAD_BACKEND)
-	set(LINK_LIBRARIES ${LINK_LIBRARIES} crashpad_client crashpad_util)
-	install(TARGETS "crashpad_handler")
```

Comment:
is this missing?

Reply:
Yes, I removed it.
But I will add it again, since it might be useful for consumers.

Reply:
In the above comment, I meant I removed the `install(TARGETS "crashpad_handler")`.
[The rest is still available](https://github.com/madebr/sentry-native/blob/77f721a870e17c121560077b21e8ba13383f896f/CMakeLists.txt#L191-L198)

Reply:
I guess you want to unconditionally install `crashpad_handler`?

Reply:
well yes. the crashpad backend is basically useless without that executable.

Reply:
This is fixed in the latest patches.